### PR TITLE
fixed error (stat panel uses created_at but select takes updated_at)

### PR DESCRIPTION
**Pull request by mohabhassan**
and changed boolean rows to show 'no' instead of ''.

### CODE REVIEW

File: src/Vendor/Laravel/Controllers/Stats.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function apiLog($uuid)
                             'is_json',
                             'wants_json',
                             'error_id',
-                            'updated_at',
```
Comment (antonioribeiro) [inline src/Vendor/Laravel/Controllers/Stats.php]: Session record is updated frequently, so created_at will have a different timestamp, but we could add created_at here

**Comment by mohabhassan** [comment]
this is the log table and not the session one.
This is page views I mean.

**Comment by mohabhassan** [comment]
I think apiVisits() is the one you're talking about.

**Comment by antonioribeiro** [comment]
Thank you!

**Pull request merged**